api/src/main/java/org/pmiops/workbench/db/dao/WorkspaceUserRoleDao.java

```diff
@@ -0,0 +1,20 @@
+package org.pmiops.workbench.db.dao;
+
+import java.util.List;
+import org.pmiops.workbench.db.model.User;
+import org.pmiops.workbench.db.model.WorkspaceUserRole;
+import org.pmiops.workbench.db.model.Workspace;
+import org.springframework.data.repository.CrudRepository;
+import org.springframework.data.jpa.repository.Query;
+
+
+/**
+ * Declaration of automatic query methods for WorkspaceUserRoles. The methods declared here are
+ * automatically interpreted by Spring Data (see README).
+ *
+ * Use of @Query is discouraged; if desired, define aliases in WorkspaceService.
+ */
+public interface WorkspaceUserRoleDao extends CrudRepository<WorkspaceUserRole, Long> {
```

Comment:
If you remove the usage above, I don't think we technically use / need this DAO anymore.

We definitely aren't using the methods explicitly defined below.

Reply:
Yeah, the methods below can go for sure. I'll look into removing the DAO to see if its possible.